cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/AnomalyNotifier.java

```diff
@@ -57,4 +57,17 @@
    * @return The old value of self healing for the given anomaly type.
    */
   boolean setSelfHealingFor(AnomalyType anomalyType, boolean isSelfHealingEnabled);
+
+  /**
+   * Get the ratio during which the self-healing is enabled over the total operating time.
+   *
+   * @return The ratio during which the self-healing is enabled over the total operating time for each anomaly type.
+   */
+  Map<AnomalyType, Float> selfHealingEnabledRatio();
+
+  /**
+   * @param nowMs Current time in ms.
+   * @return Uptime until now in ms.
+   */
+  long uptimeMs(long nowMs);
```

Comment:
The function intentionally returns uptime until the given `nowMs`, which is the current time at the beginning of some process.

Passing the time as an argument rather than computing it in the function enables users to provide a deterministic computation (i.e. not a couple `ms` off) in the calling logic. Please see `SelfHealingNotifier#selfHealingEnabledRatio(...)` for an instance of this use-case, where `nowMs` is used in a loop with a common current time and `uptime` based on this current time returned by this function.